### PR TITLE
[#2423] track core installation

### LINKED ISSUE #2423: Track Airy Core Installation
- Start Installation
- Finish Installation

some information about the kind of installation (minikube/aws)
Airy Core Version

### PR DESCRIPTION
**Pull request by pascal-airy**
Resolves #2423

### CODE REVIEW

**Review by ljupcovangelski — COMMENTED** [review]
Should we append a certain unique-id for each installation, so that we can connect it to other events in the future? Note that that is not traceable to the user or any user data, just so that we can put together various events by the same installation. Currently we generate a `unique` string with helm, so perhaps we can move that to the CLI and overwrite the helm variable. If the overwrite doesn't come - then helm should also generate this value (in case Airy is created directly with helm or Terraform and without the CLI).

**Review by ljupcovangelski — COMMENTED** [review]
What about events in case the installation is not successful?

File: cli/pkg/cmd/create/create.go [hunk 1]
```diff
@@ -49,6 +53,20 @@ func create(cmd *cobra.Command, args []string) {
...
 		return color.Colorize(color.Cyan, "#\t"+input)
 	})
+
+	client := analytics.New("7OOYJk8lQ2jfmZXKz5jJuvcK50BNDHit")
+	var rtm runtime.MemStats
+	runtime.ReadMemStats(&rtm)
+	track(client, analytics.Identify{
+		Traits: analytics.NewTraits().
+			Set("provider", providerName).
+			Set("numCpu", runtime.NumCPU()),
```
Comment (ljupcovangelski) [inline cli/pkg/cmd/create/create.go:63]: This is about the CPU on the machine that runs the CLI right? So it is only meaningful for Minikube.

File: cli/pkg/cmd/create/create.go [hunk 2]
```diff
@@ -37,6 +40,7 @@ func init() {
 	CreateCmd.Flags().StringVar(&namespace, "namespace", "default", "(optional) Kubernetes namespace that Airy should be installed to.")
 	CreateCmd.Flags().BoolVar(&initOnly, "init-only", false, "Only create the airy workspace directory and exit.")
 	CreateCmd.Flags().BoolVar(&noApply, "no-apply", false, "Don't apply any component configuration found in an existing airy.yaml file after creation.")
+	CreateCmd.Flags().BoolVar(&disableTracking, "disable-tracking", false, "Disables sending any events to Segment.")
```
Comment (chrismatix) [inline cli/pkg/cmd/create/create.go:43]:
```suggestion
	CreateCmd.Flags().BoolVar(&disableTracking, "disable-tracking", false, "Disables sending events to Segment.")
```

File: infrastructure/terraform/demos/analytics/main.tf [hunk 3]
```diff
@@ -75,7 +75,7 @@ data "template_file" "values_yaml" {
 
 resource "helm_release" "airy_core" {
   name  = "airy-release"
-  chart = "https://airy-core-helm-charts.s3.amazonaws.com/stable/airy-${replace(file("${path.module}/../../../../VERSION"), "\n", "")}.tgz"
```
Comment (chrismatix) [inline infrastructure/terraform/demos/analytics/main.tf:78]: I suppose that this should be reverted before merging

Comment (pascal-airy) [reply infrastructure/terraform/demos/analytics/main.tf]: got it 👍

File: docs/docs/cli/usage.md [hunk 4]
```diff
@@ -74,6 +74,7 @@ airy create [workspace directory] [flags]
 #### Options
 
 ```
+      --disable-tracking                 Disables sending events to Segment.
```
Comment (ljupcovangelski) [inline docs/docs/cli/usage.md:77]: Perhaps we can add sending `anonymized` events to Segment.

**Review by ljupcovangelski — APPROVED** [review]
Looks good :+1:

**Pull request merged**